casts/rows/src/types.ts

```diff
@@ -1,20 +1,23 @@
-import type { UILinkProps } from './UILink';
-import type { UICurrencyRowProps } from './UICurrencyRow';
 import type { UIAccountRowProps } from './UIAccountRow';
+import type { UICurrencyRowProps } from './UICurrencyRow';
+import type { UILinkProps } from './UILink';
+import type { UITableRowProps } from './UITableRow';
 
 export enum UIKitListRowKind {
-    Link = 'link',
-    Currency = 'currency',
     Account = 'account',
+    Currency = 'currency',
+    Link = 'link',
+    Table = 'table',
 }
 
 export type UIListRow<Kind, Props, Payload = void> = Payload extends void
     ? { kind: Kind; props: Props; key: string }
     : { kind: Kind; props: Props; key: string; payload: Payload };
 
 export type UIKitListRow<P = void> =
-    | UIListRow<UIKitListRowKind.Link, UILinkProps, P>
+    | UIListRow<UIKitListRowKind.Account, UIAccountRowProps, P>
     | UIListRow<UIKitListRowKind.Currency, UICurrencyRowProps, P>
```

Comment:
So now we have to things to use for `currency`. That's exactly why I don't like it to be in UIKit :) How one should choose proper implementation now?

Reply:
They have different appearance and use cases:
- Currency row has icon and caption (except for the title and amount) and should be used for a particular currency or token representation;
- Table row is simpler in appearance and can be used to display a list of fields of different nature (they are quite common is Surf);
As it is quite common and frequently using thing it seemed to me that including it into the UIKit system would be better neither adding some new component separately in Surf